guides/common/modules/ref_storage-requirements.adoc

```diff
@@ -38,31 +31,7 @@ endif::[]
 
 For external database servers: `/var/lib/pgsql` with installation size of 100 MB and runtime size of 20 GB.
 
-For detailed information on partitioning and size, refer to the https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/system_design_guide/partitioning-reference_system-design-guide[{RHEL} 8 partitioning guide].
-
-== [[storage-el-7]]{EL} 7
-
-.Storage Requirements for a {ProjectServer} Installation
-[cols="1,1,1",options="header"]
-|====
-|Directory |Installation Size |Runtime Size
-
-|/var/log |10 MB |10 GB
-
-|/var/opt/rh/rh-postgresql12 |100 MB |20 GB
-
-|/usr | 3 GB | Not Applicable
-
-|/opt | 3 GB | Not Applicable
-
-|/opt/puppetlabs | 500 MB | Not Applicable
-
-ifdef::katello,satellite,orcharhino[]
-|/var/lib/pulp |1 MB |300 GB
-
-|/var/lib/qpidd |25 MB | xref:storage-guidelines_{context}[Refer Storage Guidelines]
-endif::[]
-|====
+For detailed information on partitioning and size, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/system_design_guide/partitioning-reference_system-design-guide[Partitioning reference] in the _{RHEL} 8 System Design Guide_.
```

Comment:
```suggestion
For detailed information on partitioning and size, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/system_design_guide/partitioning-reference_system-design-guide[Partitioning reference] in _{RHEL} 8 System Design Guide_.
```

Reply:
In this case it should be kept because the title contains "guide".